src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -567,39 +567,31 @@ JSON example:
 
 ## Fieldmap data
 
-Data acquired to correct for B0 inhomogeneities can come in different forms. The
-current version of this standard considers four different scenarios. Please note
-that in all cases fieldmap data can be linked to a specific scan(s) it was
-acquired for by filling the IntendedFor field in the corresponding JSON file.
-For example:
+Data acquired to correct for B0 inhomogeneities can come in different forms.
+The current version of this standard considers four different scenarios.
```

Comment:
I suggest to briefly list the four scenario names already after this sentence.

The table that follows contains 5 rows and is a bit of a clash unless we are already introduced to the 4 scenarios.